syft/frameworks/torch/hook/hook_args.py

```diff
@@ -77,16 +79,30 @@
     PointerTensor: lambda i: i,
     LoggingTensor: lambda i: LoggingTensor().on(i, wrap=False),
     FixedPrecisionTensor: lambda i, **kwargs: FixedPrecisionTensor(**kwargs).on(i, wrap=False),
-    LargePrecisionTensor: lambda i, **kwargs: LargePrecisionTensor(**kwargs).on(
-        LargePrecisionTensor.create_tensor_from_numpy(i, **kwargs), wrap=False
-    ),
+    LargePrecisionTensor: lambda i, **kwargs: _lpt_backward_func(i, kwargs),
     AutogradTensor: lambda i: AutogradTensor(data=i).on(i, wrap=False),
     AdditiveSharingTensor: lambda i, **kwargs: AdditiveSharingTensor(**kwargs).on(i, wrap=False),
     MultiPointerTensor: lambda i, **kwargs: MultiPointerTensor(**kwargs).on(i, wrap=False),
     CRTPrecisionTensor: lambda i, **kwargs: CRTPrecisionTensor(**kwargs).on(i, wrap=False),
     "my_syft_tensor_type": lambda i, **kwargs: "my_syft_tensor_type(**kwargs).on(i, wrap=False)",
 }
 
+
+def _lpt_forward_func(i):
+    if isinstance(i.child, AdditiveSharingTensor):
+        return i.child
+    return i._internal_representation_to_large_ints()
```

Comment:
to be more generic, you could put the condition on the case where `_internal_representation_to_large_ints` has to be called (myabe if the child is torch.Tensor and is not a wrapper). This way, if in the future you can new syftTensor which can be a lpt.child, it won't break

Reply:
In addition, you don't have to know what a AdditiveSharingTensor is:
```
if I.child is a tensor and not a wrapper:
   return i._internal_representation_to_large_ints()
else:
    return i.child
```
```